plugin/kubernetes/ns.go

```diff
@@ -23,6 +23,8 @@ func (k *Kubernetes) nsAddr() *dns.A {
 	)
 
 	rr := new(dns.A)
+	rr.Hdr.Class = dns.ClassINET
```

Comment:
what does this fix?

Reply:
that fixes the NS record of the k8s_external.
the function nsAddr() is called by ExternalAddress(..), which provide the RR for NS. And those RR are lacking the Class and Type.